### PR TITLE
Fix commander not parsing arguments

### PR DESCRIPTION
**Pull request by ArFe**
Commander does not accept getting the options directly from program. 
Need to create options variable.

Added this line (after   .parse(process.argv) ):
  const options = program.opts();

And replace program by options thereafter.

### CODE REVIEW

File: dynamoDBtoCSV.js [hunk 1]
```diff
@@ -32,31 +32,33 @@ program
   .option("-s, --size [size]", "Number of lines to read before writing.", 5000)
   .parse(process.argv);
 
-if (!program.table) {
+  const options = program.opts();
+
+if (!options.table) {
   console.log("You must specify a table");
-  program.outputHelp();
+  options.outputHelp();
```
Comment (edasque) [inline dynamoDBtoCSV.js:39]: I don't think that's correct. options is doesn't have an outputHelp method. it's a program.OptionValue

**Comment by ArFe** [comment]
good point. Any other?

**Review by rafaelribeirokuhn — APPROVED** [review]

**Comment by rafaelribeirokuhn** [comment]
It took some time for me to figure out what was going on and fix this error. I with this PR should be merged before :(

**Comment by DollarAkshay** [comment]
Why is this not merged yet. I had the same issue and using the code in this branched worked

**Review by DollarAkshay — APPROVED** [review]

**Comment by houmie** [comment]
Why has this not been merged yet? This project is not functional without this.

**Pull request merged**